test/e2e/framework/util.go

```diff
@@ -1934,6 +1937,71 @@ func (config *RCConfig) applyTo(template *api.PodTemplateSpec) {
 	}
 }
 
+type RCStartupStatus struct {
+	Expected              int
+	Terminating           int
+	Running               int
+	RunningButNotReady    int
+	Waiting               int
+	Pending               int
+	Unknown               int
+	Inactive              int
+	FailedContainers      int
+	Created               []*api.Pod
+	ContainerRestartNodes sets.String
+}
+
+func PrintPodStartupStatus(name string, startupStatus RCStartupStatus) {
```

Comment:
This should be a method on RCStartupStatus, i.e.

func (s RCStartupStatus) Print(name string) { ... }